Client/Frontend/Browser/TabManager.swift

```diff
@@ -346,7 +346,8 @@ class TabManager: NSObject {
             return
         }
 
-        tabs.swapAt(fromIndex, toIndex)
+        let tab = tabs.remove(at: fromIndex)
+        tabs.insert(tab, at: toIndex)
```

Comment:
Is this so it properly selects the tab? If so, should leave a comment _why_ we aren't using the more "intuitive" `swapAt`.

Reply:
No this is because using `swapAt` is inherently wrong to use in this case because the UI doesn't swap two tabs, it only moves a single tab and the others are moved based on the new location of that tab.  I don't think we need a comment to use the correct logic :)